src/script/components/UserDevices.tsx

```diff
@@ -0,0 +1,363 @@
+                key={client.id}
+                className={cx('participant-devices__device-item', {
+                  'participant-devices__device-item--padding': !noPadding,
+                })}
+                data-uie-name="item-device"
+              >
+                <DeviceCard device={client} click={() => clickOnDevice(client)} showVerified showIcon />
+              </div>
+            ))}
+          </div>
+        </>
+      )}
+
+      {showDeviceList && deviceMode === FIND_MODE.NOT_FOUND && (
+        <div className={cx('participant-devices__header', {'participant-devices__header--padding': !noPadding})}>
+          <div className="participant-devices__text-block panel__info-text" data-uie-name="status-devices-headline">
+            {userEntity
+              ? t('participantDevicesOutdatedClientMessage', {
+                  brandName: Config.getConfig().BRAND_NAME,
+                  user: userName,
+                })
+              : ''}
+          </div>
+          <a
+            className="participant-devices__link accent-text"
+            href={getPrivacyPolicyUrl()}
+            rel="nofollow noopener noreferrer"
+            target="_blank"
+          >
+            {t('participantDevicesLearnMore')}
+          </a>
+        </div>
+      )}
+
+      {current.state === UserDevicesState.DEVICE_DETAILS && (
+        <div className={cx('participant-devices__header', {'participant-devices__header--padding': !noPadding})}>
+          <div
+            className="participant-devices__link participant-devices__show-self-fingerprint accent-text"
+            onClick={clickToShowSelfFingerprint}
+          >
+            {t('participantDevicesDetailShowMyDevice')}
+          </div>
+          <div
+            className="panel__info-text"
+            dangerouslySetInnerHTML={{
+              __html: userEntity ? t('participantDevicesDetailHeadline', {user: userName}) : '',
+            }}
+          ></div>
+          <a
+            className="participant-devices__link accent-text"
+            href={getPrivacyHowUrl()}
+            rel="nofollow noopener noreferrer"
+            target="_blank"
+          >
+            {t('participantDevicesDetailHowTo')}
+          </a>
+          <div className="participant-devices__single-client">
+            <DeviceCard device={selectedClient} />
+          </div>
+          <div className="participant-devices__fingerprint" data-uie-name="status-fingerprint">
+            {fingerprintRemote.map((part, index) => (
+              <React.Fragment key={`${index}-${part}`}>
+                <span className="participant-devices__fingerprint__part">{part}</span>{' '}
+              </React.Fragment>
+            ))}
+          </div>
+
+          <div className="participant-devices__verify">
+            <div className="slider" data-uie-name="do-toggle-verified">
+              <input className="slider-input" type="checkbox" name="toggle" id="toggle" defaultChecked={isVerified} />
```

Comment:
Mh, I found `GuestModeToggle.tsx` in the existing code and wonder if it would make sense to extract a toggle component. Could also be a separate PR.